learning/web/content/modules/benzodiazepines/CON234573_21.mdx

```diff
@@ -15,15 +15,14 @@ import Expander from '../../../src/components/Expander'
 * It is alright to drink alcohol socially with this medicine
 **Choose your answers as follows:**
 
-* If i, ii and iii are correct
-* If ii and iv are correct
-* If only v is correct
-* If i, ii, iii and v are correct
+* If 1, 2 and 3 are correct
+* If 2 and 4 are correct
+* If only 5 is correct
+* If 1, 2, 3 and 5 are correct
 * If all are correct
 [**Feedback on the responses is given at the end of the module**](/benzodiazepines/CON234573_22#q1)
```

Comment:
is there a missing line break here?

Reply:
Yes; I haven't got up to Benzodiazepines yet. I just did a search and replace on the changed phrases above.